README.md

```diff
@@ -23,6 +23,15 @@ Anyone is welcome to contribute to Apollo Server, just read [CONTRIBUTING.md](./
 
 Apollo Server is super-easy to set up. Just npm-install apollo-server, write a GraphQL schema, and then use one of the following snippets to get started. For more info, read the [Apollo Server docs](http://docs.apollostack.com/apollo-server).
 
+### TypeScript project
+
+To build your GraphQL server using TypeScript, Apollo Server is the project for you.  To add Apollo to your project:
+
+```sh
+npm install apollo-server
+typings i -SG dt~express dt~express-serve-static-core dt~serve-static dt~mime dt~hapi dt~cookies dt~koa
+```
+
```

Comment:
If we make Typescript this prominent, we should call this section "installing in TypeScript" and create one just above it called "Installing in JavaScript". I think there will still be more people using apollo server with JavaScript, and I wouldn't want them to get confused and think that they can only use it with Typescript.